XCUITests/NewTabSettings.swift

```diff
@@ -30,6 +30,23 @@ class NewTabSettingsTest: BaseTestCase {
         waitForNoExistence(app.collectionViews.cells["TopSitesCell"].collectionViews.cells["youtube"])
         waitForNoExistence(app.staticTexts["Highlights"])
     }
+    
+    func testChangeNewTabSettingsShowFirefoxHome() {
+        // Set to blank page first since FF Home is default
+        navigator.goto(NewTabSettings)
+        navigator.performAction(Action.SelectNewTabAsBlankPage)
+        navigator.performAction(Action.OpenNewTabFromTabTray)
+        waitForNoExistence(app.collectionViews.cells["TopSitesCell"])
```

Comment:
Don't forget to modify this if you use a different New Tab. Also, I would say not to use the watiForNoExistence in this case because if topsites does not appear it does not imply that you have opened a blank page..it could be the bookmark or history...so I would go for asserting the existence of a element that we know it should be there in the correct screen...hope this makes sense :)